inference/generativeai/llm-workshop/lab7-openassistant-sft-12b/oasst-sft-4-pythia-12b-sagemaker.ipynb

```diff
@@ -0,0 +1,1156 @@
+{
```

Comment:
FI, I have edited the License agreement section and specified that both the model and the dataset it has been trained on are under the Apache 2.0 license.

Reply:
<p>So, do you have a ticket from legal explicitly approving it? Seems like legal needs to comb through it themselves to make sure we don't miss any subtleties (i.e. model may be fine but data is not). If you don't have a ticket, you submit one at https://t.corp.amazon.com/create/templates/b9c3204f-49a4-4740-adb3-cccb0b4a4d9f</p><p> </p><p>Also, we need to put this model in our designated notebooks bucket (sagemaker-sample-files) so that we don't have a broken link in the future. Once you get legal approval, it is quick and easy to upload (I can help with that), and we should be good to go</p>